src/main/java/org/tron/common/runtime/vm/program/invoke/ProgramInvokeImpl.java

```diff
@@ -150,7 +153,7 @@ public DataWord getCallValue() {
   /* NOTE: In the protocol there is no restriction on the maximum message data,
    * However msgData here is a byte[] and this can't hold more than 2^32-1
    */
-  private static BigInteger MAX_MSG_DATA = BigInteger.valueOf(Integer.MAX_VALUE);
+  private static final BigInteger MAX_MSG_DATA = BigInteger.valueOf(Integer.MAX_VALUE);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this variable to comply with Java Code Conventions. ([squid:S1213](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1213))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHR-SgN4uCcljsr5&open=AWbnEHR-SgN4uCcljsr5)

[ISSUE_KEY]: AWbnEHR-SgN4uCcljsr5